cloud-chatops/cron-clone.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# When copying the contents of this file, remember to name the new file using only valid characters [a-zA-Z0-9_-]
+# Fullstops '.' and file extenstions are not permitted
+
+set -euo pipefail -x
+
+git fetch && git clean -fxd && git reset --hard origin/main
+
+cd $HOME/cloud-docker-images/cloud-chatops
```

Comment:
Unfortunately this won't work as the order is back-to-front
We need to move to the git repo then update it, whereas the script attempts to update the repo then moves to it

Reply:
Ah yes that makes sense, `cd $HOME/cloud-docker-images/cloud-chatops` was actually originally only intended for the line below, `docker compose up -d`. Before the last change, we were deleting the repo with an absolute path, therefore there was no need to `cd` into it. I can get that changed now, thanks!